cli_reference/openshift_developer_cli/odo-release-notes.adoc

```diff
@@ -6,32 +6,48 @@ include::modules/common-attributes.adoc[]
 toc::[]
 
 [id="odo-notable-improvements_{context}"]
-== Notable improvements in `{odo-title}`{nbsp}{odo-ver}
+== Notable changes and improvements in `{odo-title}`{nbsp}{odo-ver}
+
+* A `--now` flag is added for `odo create`, `odo url` and `odo config`. 
++
+Run `odo url create --now` to create the URL in the configuration and propagate all the changes to the cluster
+* An `odo debug info` command is added. The command displays the information whether the debug mode is enabled for the component, whether the port-forward process is running, and information about ports used.
+* `odo push` output now shows the exact error from the OpenShift cluster on failure.
++
+* A `--secure` flag is enabled for `odo url`. It indicates whether a URL is secure. 
```

Comment:
s/enabled/added ?
To match _A `--now` flag is added_
 https://github.com/openshift/openshift-docs/pull/20957/files#diff-c2001e8b2095b3f6a515b452c5c694f1R11